5m5v-config.yaml.example

```diff
@@ -1,7 +1,3 @@
-consumer_key: Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0A
-consumer_secret: Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa
 users:
-    - access_token: 0000000000000000000-Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0
-      access_token_secret: Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0
-      filters:
-          - english
+  - api_key: Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0Aa0
+    language: english

```

Comment:
Updated the config to only handle one language per user, for simplicity (we don't need several).